lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -717,6 +717,21 @@ def cmd_up(self):
             result['changed'] = build_output['changed']
             result['actions'] += build_output['actions']
 
+        if self.remove_orphans:
+            containers = self.client.containers(
+                filters={'label': ['{0}={1}'.format(LABEL_PROJECT, self.project.name), '{0}={1}'.format(LABEL_ONE_OFF, "False")]}
```

Comment:
```suggestion
                filters={
                    'label': ['{0}={1}'.format(LABEL_PROJECT, self.project.name),
                    '{0}={1}'.format(LABEL_ONE_OFF, "False")],
                }
```
Makes this easier to read.